cmd/entrypoint-wrapper/main_test.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/openshift/ci-tools/pkg/api"
+)
+
+func TestManageCLI(t *testing.T) {
+	for _, testCase := range []struct {
+		testName string
+		cliDir   string
+		expected string
+	}{
+		{
+			testName: "If CLI directory is set, append it to the PATH environment variable",
+			cliDir:   "/usr/local/bin/oc",
+			expected: os.Getenv("PATH") + ":/usr/local/bin/oc",
+		},
+		{
+			testName: "If CLI directory is not set, leave PATH unchanged",
+			expected: os.Getenv("PATH"),
+		},
+	} {
+		t.Run(testCase.testName, func(t *testing.T) {
+			originalPath := os.Getenv("PATH")
+
+			defer os.Setenv("PATH", originalPath)
+
+			cmd := exec.Command("echo")
+			cmd.Env = os.Environ()
+
+			if testCase.cliDir != "" {
+				os.Setenv(api.CliEnv, testCase.cliDir)
+				defer os.Unsetenv(api.CliEnv)
+			}
+
+			manageCLI(cmd)
+
+			gotPath := ""
+			for _, env := range cmd.Env {
+				if strings.HasPrefix(env, "PATH=") {
+					gotPath = strings.TrimPrefix(env, "PATH=")
+				}
+			}
+
+			if gotPath != testCase.expected {
+				t.Fatalf("expected PATH to be %q, got %q", testCase.expected, gotPath)
+			}
+		})
+	}
+}
+
+func TestManageHome(t *testing.T) {
+	fileModePtr := func(fm os.FileMode) *os.FileMode { return &fm }
+
+	for _, testCase := range []struct {
+		testName string
+		homeSet  bool
+		wantEnv  []string
+		wantHome string
```

Comment:
expectedEnv or expectedHome
I would delete one of it, they are basically the same